### PR TITLE
Updates codelabs to alpha03

### PR DESCRIPTION
**Pull request by manuelvicnt**

### CODE REVIEW

File: StateCodelab/finished/src/main/java/com/codelabs/state/todo/TodoScreen.kt [hunk 1]
```diff
@@ -21,19 +21,8 @@ import androidx.compose.foundation.Icon
...
-import androidx.compose.foundation.layout.IntrinsicSize
-import androidx.compose.foundation.layout.Row
-import androidx.compose.foundation.layout.Spacer
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.height
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.preferredHeight
-import androidx.compose.foundation.layout.width
-import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.layout.*
```
Comment (nickbutcher) [inline StateCodelab/finished/src/main/java/com/codelabs/state/todo/TodoScreen.kt:24]: Remove * import

File: ThemingCodelab/build.gradle [hunk 2]
```diff
@@ -15,22 +15,31 @@
...
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.2.0-alpha08"
+        classpath "com.android.tools.build:gradle:4.2.0-alpha10"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+
+        // NOTE: Do not place your application dependencies here; they belong
```
Comment (nickbutcher) [inline ThemingCodelab/build.gradle:30]: Remove comment

**Review by nickbutcher — APPROVED** [review]
Changes to Theming Codelab LG. Thanks!

File: StateCodelab/start/src/main/java/com/codelabs/state/todo/TodoScreen.kt [hunk 3]
```diff
@@ -19,12 +19,7 @@ package com.codelabs.state.todo
 import androidx.compose.foundation.Icon
 import androidx.compose.foundation.Text
 import androidx.compose.foundation.clickable
-import androidx.compose.foundation.layout.Arrangement
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.InnerPadding
-import androidx.compose.foundation.layout.Row
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.*
```
Comment (JoseAlcerreca) [inline StateCodelab/start/src/main/java/com/codelabs/state/todo/TodoScreen.kt:22]: ICYM

**Review by JoseAlcerreca — APPROVED** [review]

**Comment by objcode** [comment]
State codelab LGTM w/o *

**Review by objcode — APPROVED** [review]

**Review by chrisbanes — APPROVED** [review]

**Pull request merged**